src/api/app/types.ts

```diff
@@ -194,12 +262,26 @@ export type NetworkInfoResponse = NetworkInfo[];
 /**
  * @public
  */
-export interface AppApi {
-  appInfo: Requester<AppInfoRequest, AppInfoResponse>;
-  callZome: Requester<CallZomeRequest, CallZomeResponse>;
-  enableCloneCell: Requester<EnableCloneCellRequest, EnableCloneCellResponse>;
-  disableCloneCell: Requester<
-    DisableCloneCellRequest,
-    DisableCloneCellResponse
-  >;
+export interface AppAgentClient {
```

Comment:
I think we can rename all `AppAgent*` to `App*`. WDYT?

Reply:
Heck, had moved this stuff without looking too closely - hadn't realised how many instances there were of 'agent' still. On it!